### ISSUE TITLE
this.platform.is('tablet') not working properly

### ISSUE THREAD

**Issue by smxdevst** [ionic-team/ionic#15434]
# Bug Report

```
Ionic:

   ionic (Ionic CLI)  : 4.1.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.11

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-webview 1.2.1, (and 25 other plugins)

System:

   ios-deploy : 1.9.2
   NodeJS     : v8.9.4 (/usr/local/bin/node)
   npm        : 6.0.1
   OS         : macOS High Sierra
   Xcode      : Xcode 9.4.1 Build version 9F2000
```

**Describe the Bug**
On my phone Google Pixel 2 XL `this.platform.is('tablet')` giving true

**Steps to Reproduce**
Add `this.platform.is('tablet')` on any page on Google Pixel 2 XL

**Expected Behavior**
`this.platform.is('tablet')` should return false.

**Comment by yuankunluo** [comment]
The platforms are: Android,iOS, windows, browser, windowsphone

**Comment by smxdevst** [comment]
@yuankunluo 

https://ionicframework.com/docs/api/platform/Platform/

![d0ddu5xe4b8](https://user-images.githubusercontent.com/13336971/44960517-fe167b80-aeb5-11e8-8310-f579091f03f0.jpg)

**Comment by daem0ndev** [comment]
Here’s another resource I found for detecting pixel 2 xl: https://stackoverflow.com/questions/49078882/media-query-to-differentiate-google-pixel-1-2-vs-google-pixel-xl-vs-google-pixel

**Comment by liamdebeasi** [comment]
Hi there,

This issue has been resolved via #17630 and will be in the next release of Ionic.

Thanks for the issue!
